spec/models/classification_pipeline_spec.rb

```diff
@@ -123,7 +123,7 @@
     expect(Reduction.count).to eq(2)
     expect(Reduction.where(subgroup: 1).first.data).to include({"LN" => 2, "TGR" => 1})
     expect(Reduction.where(subgroup: 2).first.data).to include({"LN" => 2, "BR" => 1})
-    expect(Reduction.where(subgroup: 1).first.data.keys).not_to include("classroom")
+    # expect(Reduction.where(subgroup: 1).first.data.keys).not_to include("classroom")
```

Comment:
is this meant to be commented? remove if the expectation isn't used..

Reply:
meant to just remove this, good catch